src/lean_spec/subspecs/containers/block/block.py

```diff
@@ -56,14 +57,29 @@ class Block(Container):
     """The block's payload."""
 
 
-class SignedBlock(Container):
-    """A container for a block and the proposer's signature."""
+class BlockAndVote(Container):
```

Comment:
should we go with ...Vote or ..Attestation nomenclature here @syjn99 @unnawut

Attestation could confuse with attestations in the block body

Reply:
I think there will be questionings here once we start to form production specs, but I'm more than happy to keep going with Vote for single, non-aggregated vote and Attestation for the aggregation. The distinction helps with readability here.

Reply:
I just re-read this PR's `vote.py` again. I strongly prefer `Vote` & `Attestation` over `ValidatorAttestation` & `Attestation`

Reply:
We could use `AggregatedAttestations` for aggregated attestations. This is straightforward and is in line with `AggregatedSignatures`.

Reply:
ok lets do this:

ValidatorAttestation -> Attestation
SignedValidationAttestation -> SignedAttestation

Attestation -> AggreagtedAttestation
SignedAttestation -> SignedAggreagtedAttestation

cc @GrapeBaBa

Reply:
To avoid any misunderstanding, this is `AggreagtedAttestations` and `SignedAggreagtedAttestations`, right?